src/BottomNavigation/BottomNavigation.js

```diff
@@ -0,0 +1,73 @@
+// @flow weak
+
+import React, { PropTypes, Children, cloneElement } from 'react';
+import { createStyleSheet } from 'jss-theme-reactor';
+import classNames from 'classnames';
+
+export const styleSheet = createStyleSheet('BottomNavigation', (theme) => {
```

Comment:
@nathanmarks What politics should we have with `MUI_SHEET_ORDER`? I'm wondering if we need to add it there.